cli/cmd/agent_install_ec2ic.go

```diff
@@ -20,8 +20,9 @@ package cmd
 
 import (
 	"fmt"
+	"sync"
 
-	"github.com/pkg/errors"
+	"github.com/korovkin/limiter"
```

Comment:
why adding this MIT Go package rather than implementing the parallelism ourselves?

Reply:
yeah,lets move this back to Slack 😸

Reply:
Resolve this with 4ae6efdfe27251ff26b4041fa07cdea55229d048